src/lib/WalletInfoCollector.ts

```diff
@@ -30,6 +30,7 @@ export type WalletCollectionResult = {
 export default class WalletInfoCollector {
     public static async collectWalletInfo(
         walletType: WalletType,
+        keyId: string,
         walletId?: string,
```

Comment:
If I'm not mistaken, this parameter is not needed anymore as it is always undefined.

Reply:
I think so, too.